index.html

```diff
@@ -119,100 +119,292 @@ <h3 class="work-name">Tonic</h3>
               <li class="lang-section">css</li>
               <li class="lang-section">javascript</li>
             </ul>
-            <button type="button" class="work-btn">See Project</button>
+            <button type="button" data-modal-target="#modal1" class="work-btn">
+              See Project
+            </button>
           </div>
-        </li>
+          <div class="modal" id="modal1">
+            <div class="modal-header">
+              <div class="title">Tonic</div>
+
+              <button type="button" data-close-button class="close-button">&times;</button>
+            </div>
 
+            
+            <ul class="modal-info">
+              <li class="info-modal">CANOPY</li>
+              <li><i class="info-sep fa fa-circle"></i></li>
+              <li class="info-modal">Back End Dev &nbsp;</li>
+                <li><i class="info-sep fa fa-circle"></i></li>
+                <li class="info-description info-modal">2015</li>
+              </ul>
+              
+              <div class="modal-body">
+                <img
+                src="img/work-img/work-preview1.png"
```

Comment:
- [ ] On the popup window, the project image is broken, I think it would be very nice if you fixed it

Reply:
Well addressed this!